### ISSUE TITLE
Reproduce result for Kitti benchmark

### ISSUE THREAD

**Issue by nghiattran** [MarvinTeichmann/KittiBox#31]
@MarvinTeichmann How reproduce results in your Multinet paper? I tried to add resnet decoders and compare them with the VGG decoder but when I submit results onto http://www.cvlibs.net/datasets/kitti/, it turns out that VGG can only reach 50% on moderate. I used `min_conf=0.50, tau=0.35` and exactly same setting as in provided pretrained VGG.

**Comment by MarvinTeichmann** [comment]
Strange. What is your validation score?

**Comment by nghiattran** [comment]
Score produced by `tv-analyze`:

|           	| train easy 	| train moderate 	| train hard 	| val easy 	| val moderate 	| val hard 	| speed (msec) 	| speed (fps) 	| post (msec) 	|
|-----------	|------------	|----------------	|------------	|----------	|--------------	|----------	|--------------	|-------------	|-------------	|
| VGG       	| 93.25%     	| 84.10%         	| 69.09%     	| 94.27%   	| 86.32%       	| 70.78%   	| 104.4554     	| 9.5735      	| 3.5804      	|

**Comment by robin2002** [comment]
I also submitted a result using the model (KittiBox_pretrained.zip), which is provided by author (MarvinTeichmann) and got the 45.22% for the moderate in a car detection task. (Easy: 71.99%, Hard: 36.83%) . Have you produced results using the Kitti test dataset?

**Comment by nghiattran** [comment]
This is the result for the test set. This result is higher than what I originally got but it is very quite low compared to what is in the official paper.

![screenshot from 2017-06-01 00-09-32](https://cloud.githubusercontent.com/assets/13736109/26664351/d9a06c40-465e-11e7-9086-b503c394ac45.png)
